### ISSUE TITLE
Add POPM tag for Windows

### ISSUE THREAD

**Issue by SaiyanRiku** [clementine-player/Clementine#5636]
Clementine is tagging mp3 rating within the file, with the POPM frame, which is very great.

It could be better if Clementine also add the Windows Media Player POPM frame.
The frame is looking like this:
POPM=Windows Media Player 9 Series=None 196/255

This will allow to get the same rating in Windows Explorer than in the Clementine Library.

Note Windows Explorer only support ID3v2.3 and may need to fix this issue #5146 to get everything working.

**Comment by gordon-morehouse** [comment]
Are multiple POPM frames even officially supported? I don't believe so but I could be wrong.

This introduces the possibility of inducing bugs in other players and of multiple POPM entries in a given file becoming uncoupled and thus out of sync, all because WMP is apparently not capable of reading POPM entries from other players.

**Comment by SaiyanRiku** [comment]
Yes you are right. I tested it, and i see that WMP cannot read it own POPM frame if there are multiple POPM frame in the file. So there is no way to get it working with both.

**Comment by SaiyanRiku** [comment]
Or if possible add an option to read/write the POPM frame in the WMP format.

**Comment by gordon-morehouse** [comment]
Yeah, I actually have an interest in the POPM frame which is why I took a look at this bug. Having read up on it, support varies among players. The first part of the frame was intended to be an email address for the user who rated the song, but most players just write a fixed string - the player name or similar - and that's it.

**Comment by SaiyanRiku** [comment]
Yes, that suck there is no way to get rating working easily on multiple player.
